resotocore/resotocore/db/graphdb.py

```diff
@@ -155,7 +155,7 @@ async def create_update_schema(self) -> None:
         pass
 
     @abstractmethod
-    async def copy_graph(self, to_graph: GraphName) -> "GraphDB":
+    async def copy_graph(self, to_graph: GraphName, snapshot: bool = False) -> "GraphDB":
```

Comment:
Can we give it a more precise name? Reading it this way, it is not clear if I should set it when the graph is a snapshot or copied to a snapshot.